bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/BridgeImpl.java

```diff
@@ -24,7 +24,10 @@
 public class BridgeImpl extends ThingImpl implements Bridge {
 
     private transient List<Thing> things = new CopyOnWriteArrayList<>();
-    private final Logger logger = LoggerFactory.getLogger(BridgeImpl.class);
+
+    private static Logger logger() {
```

Comment:
IIRC Gson is using sun.misc.Unsafe to create object instances on de-serialization. I don't know if if there is any specification how this function -- that intention is to be used only by the JVM itself and not by external stuff -- initialize the fields using fields initializers. Or at least how the Unsafe object creation works at all.
That's the reason I don't use the transient keyword, because I know this method will work all the time. On the other one, I can just try if it is working currently on my machine, but I don't know about the spec.